website/blog/2024-09-24-run-ferretdb-cybertec-postgres-kubernetes.md

```diff
@@ -0,0 +1,350 @@
+ferretdb-8c5468db-mmqtr              1/1     Running   0          38s
+postgres-operator-78d4fdc97b-mp49x   1/1     Running   0          18m
+```
+
+You can also view the service by running `kubectl get svc -n cpo`.
+
+The output should look like this:
+
+```text
+% kubectl get svc -n cpo
+NAME                    TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)     AGE
+cluster-1               ClusterIP   10.105.156.194   <none>        5432/TCP    36m
+cluster-1-clusterpods   ClusterIP   None             <none>        <none>      36m
+cluster-1-repl          ClusterIP   10.101.77.119    <none>        5432/TCP    36m
+ferretdb-service        ClusterIP   10.102.252.125   <none>        27017/TCP   19m
+```
+
+## Access FerretDB instance via `mongosh`
+
+You can connect to the FerretDB instance using `mongosh`.
+
+Start by creating a temp `mongosh` pod:
+
+```sh
+kubectl run -it --rm --image=mongo:latest mongo-client -- bash
+```
+
+Connect to the FerretDB instance using the Postgres credentials generated earlier:
+
+```sh
+mongosh "mongodb://postgres:<password>@host:27017/postgres?authMechanism=PLAIN"
+```
+
+Ensure to use the password generated from your the user credential in `Secret`.
+The host is the cluster IP of the FerretDB service (e.g. `10.102.252.125`).
+
+## Run CRUD operations on FerretDB
+
+Let's run some CRUD commands to see how FerretDB enables you to replace MongoDB and run your familiar queries and operations.
+
+Start by inserting some documents into a `weather` collection:
+
+```js
+db.weather.insertOne([
+  {
+    date: new Date('2024-04-22'),
+    location: {
+      city: 'New York',
+      country: 'USA',
+      coordinates: { lat: 40.7128, lon: -74.006 }
+    },
+    weather: {
+      temperature: 18,
+      conditions: 'Cloudy',
+      wind_speed: 12,
+      humidity: 80
+    },
+    remarks: 'Possible light rain in the evening.'
+  }
+])
+```
+
+Query the collection to see the inserted document:
+
+```js
+db.weather.find()
+```
+
+Run an update operation to update the humidity of the document:
+
+```js
+db.weather.updateMany(
+  { 'location.city': 'New York', 'weather.wind_speed': { $gt: 10 } },
+  { $set: { 'weather.humidity': 85 } }
+)
+```
+
+Run `db.weather.find()` on the collection to see the updated document:
+
+```json5
+[
+  {
+    _id: ObjectId('66f008484f7a5c7f5a1681ed'),
+    date: ISODate('2024-04-22T00:00:00.000Z'),
+    location: {
+      city: 'New York',
+      country: 'USA',
+      coordinates: { lat: 40.7128, lon: -74.006 }
+    },
+    weather: {
+      temperature: 18,
+      conditions: 'Cloudy',
+      wind_speed: 12,
+      humidity: 85
+    },
+    remarks: 'Possible light rain in the evening.'
+  }
+]
+```
+
+## View data in Postgres via `psql`
+
+FerretDB stores the data in a JSONB format in the Postgres database.
+If you want to know how this looks, connect to the database via `psql` using the database and user credentials:
+
+```sh
+PGPASSWORD=<password> psql -h 127.0.0.1 -p 5432 -U postgres
```

Comment:
I'm a bit confused. Postgresql is not running on localhost (i.e. `127.0.0.1`). Where do you meant to run psql command :thinking: ?